NPE if profile is displayed with signed lincense of a file which has been deleted

I went to profile selected the item I had signed license for and deleted that item...
![image](https://cloud.githubusercontent.com/assets/1842385/8750528/34936832-2ca8-11e5-9153-d7fa01785fb9.png)
```
org.apache.cocoon.ProcessingException: Could not read resource cocoon://DRI/5/admin/item
        at <map:serialize type="xml"> - resource://aspects/EPerson/sitemap.xmap:370:31
        at <map:transform type="Navigation"> - resource://aspects/EPerson/sitemap.xmap:130:38
        at <map:generate> - resource://aspects/EPerson/sitemap.xmap:119:19
        at <map:serialize type="xml"> - resource://aspects/Submission/sitemap.xmap:263:38
        at <map:transform type="Navigation"> - resource://aspects/Submission/sitemap.xmap:114:45
        at <map:generate> - resource://aspects/Submission/sitemap.xmap:111:26
        at <map:serialize type="xml"> - resource://aspects/Statistics/sitemap.xmap:359:36
        at <map:generate> - resource://aspects/Statistics/sitemap.xmap:39:19
        at <map:serialize type="xml"> - resource://aspects/Workflow/sitemap.xmap:164:38
        at <map:generate> - resource://aspects/Workflow/sitemap.xmap:79:26
        at <map:serialize type="xml"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/aspects/aspects.xmap:85:34
        at <map:transform type="PageNotFound"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/aspects/aspects.xmap:84:43
        at <map:generate> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/aspects/aspects.xmap:83:22
        at <map:serialize type="xhtml"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:150:34
        at <map:transform type="i18n"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:144:33
        at <map:transform> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:139:51
        at <map:transform type="IncludePageMeta"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:127:55
        at <map:generate type="file"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:120:55
        at <map:match> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/UFAL/sitemap.xmap:117:28
        at <map:mount> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/themes.xmap:33:45
        at <map:match type="ThemeMatcher"> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/themes/themes.xmap:32:35
        at <map:mount> - file:///mnt/DATA/121/home/dspace/installations/dspace/upgrade_test/webapps/xmlui/sitemap.xmap:751:94
        at org.apache.cocoon.components.source.util.SourceUtil.handleSAXException(SourceUtil.java:409)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:138)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:750)
        at sun.reflect.GeneratedMethodAccessor367.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy109.process(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.toSASitemapSource.java:362)
        at org.apache.cocoon.components.source.util.SourceUtil.toSASourceUtil.java:111)
        at org.apache.cocoon.components.source.util.SourceUtil.parse(SourceUtil.java:294)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:136)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.processXMLPipeline(AbstractCachingProcessingPipeline.java:273)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:750)
        at sun.reflect.GeneratedMethodAccessor367.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.process(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.toSASitemapSource.java:362)
        at org.apache.cocoon.components.source.util.SourceUtil.toSASourceUtil.java:111)
        at org.apache.cocoon.components.source.util.SourceUtil.parse(SourceUtil.java:294)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:136)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.processXMLPipeline(AbstractCachingProcessingPipeline.java:273)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:750)
        at sun.reflect.GeneratedMethodAccessor367.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.process(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.toSASitemapSource.java:362)
        at org.apache.cocoon.components.source.util.SourceUtil.toSASourceUtil.java:111)
        at org.apache.cocoon.components.source.util.SourceUtil.parse(SourceUtil.java:294)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:136)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.processXMLPipeline(AbstractCachingProcessingPipeline.java:273)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:750)
        at sun.reflect.GeneratedMethodAccessor367.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.process(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.toSASitemapSource.java:362)
        at org.apache.cocoon.components.source.util.SourceUtil.toSASourceUtil.java:111)
        at org.apache.cocoon.components.source.util.SourceUtil.parse(SourceUtil.java:294)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:136)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.processXMLPipeline(AbstractCachingProcessingPipeline.java:273)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:750)
        at sun.reflect.GeneratedMethodAccessor367.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.process(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.toSASitemapSource.java:362)
        at org.apache.cocoon.components.source.util.SourceUtil.toSASourceUtil.java:111)
        at org.apache.cocoon.components.source.util.SourceUtil.parse(SourceUtil.java:294)
        at org.apache.cocoon.generation.FileGenerator.generate(FileGenerator.java:136)
        at sun.reflect.GeneratedMethodAccessor366.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.generate(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.processXMLPipeline(AbstractProcessingPipeline.java:544)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.processXMLPipeline(AbstractCachingProcessingPipeline.java:273)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:439)
        at sun.reflect.GeneratedMethodAccessor333.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.process(Unknown Source)
        at org.apache.cocoon.components.treeprocessor.sitemap.SerializeNode.invoke(SerializeNode.java:147)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:55)
        at org.apache.cocoon.components.treeprocessor.sitemap.MatchNode.invoke(MatchNode.java:87)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelineNode.invoke(PipelineNode.java:143)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelinesNode.invoke(PipelinesNode.java:81)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:239)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:171)
        at org.apache.cocoon.components.treeprocessor.TreeProcessor.process(TreeProcessor.java:247)
        at org.apache.cocoon.components.treeprocessor.sitemap.MountNode.invoke(MountNode.java:117)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:55)
        at org.apache.cocoon.components.treeprocessor.sitemap.MatchNode.invoke(MatchNode.java:87)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelineNode.invoke(PipelineNode.java:143)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelinesNode.invoke(PipelinesNode.java:81)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:239)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:171)
        at org.apache.cocoon.components.treeprocessor.TreeProcessor.process(TreeProcessor.java:247)
        at org.apache.cocoon.components.treeprocessor.sitemap.MountNode.invoke(MountNode.java:117)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelineNode.invoke(PipelineNode.java:143)
        at org.apache.cocoon.components.treeprocessor.AbstractParentProcessingNode.invokeNodes(AbstractParentProcessingNode.java:78)
        at org.apache.cocoon.components.treeprocessor.sitemap.PipelinesNode.invoke(PipelinesNode.java:81)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:239)
        at org.apache.cocoon.components.treeprocessor.ConcreteTreeProcessor.process(ConcreteTreeProcessor.java:171)
        at org.apache.cocoon.components.treeprocessor.TreeProcessor.process(TreeProcessor.java:247)
        at org.apache.cocoon.servlet.RequestProcessor.process(RequestProcessor.java:351)
        at org.apache.cocoon.servlet.RequestProcessor.service(RequestProcessor.java:169)
        at org.apache.cocoon.sitemap.SitemapServlet.service(SitemapServlet.java:84)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.apache.cocoon.servletservice.ServletServiceContext$PathDispatcher.forward(ServletServiceContext.java:468)
        at org.apache.cocoon.servletservice.ServletServiceContext$PathDispatcher.forward(ServletServiceContext.java:443)
        at org.apache.cocoon.servletservice.spring.ServletFactoryBean$ServiceInterceptor.invoke(ServletFactoryBean.java:264)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:202)
        at com.sun.proxy.$Proxy100.service(Unknown Source)
        at org.dspace.springmvc.CocoonView.render(CocoonView.java:117)
        at org.springframework.web.servlet.DispatcherServlet.render(DispatcherServlet.java:1180)
        at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:950)
        at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:852)
        at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:882)
        at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:789)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:644)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.dspace.app.xmlui.cocoon.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:111)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.dspace.app.xmlui.cocoon.DSpaceCocoonServletFilter.doFilter(DSpaceCocoonServletFilter.java:276)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.dspace.app.xmlui.cocoon.servlet.multipart.DSpaceMultipartFilter.doFilter(DSpaceMultipartFilter.java:119)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.dspace.utils.servlet.DSpaceWebappServletFilter.doFilter(DSpaceWebappServletFilter.java:78)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:501)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
        at com.googlecode.psiprobe.Tomcat80AgentValve.invoke(Tomcat80AgentValve.java:36)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:610)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:516)
        at org.apache.coyote.ajp.AbstractAjpProcessor.process(AbstractAjpProcessor.java:831)
        at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:659)
        at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.doRun(AprEndpoint.java:2431)
        at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.run(AprEndpoint.java:2420)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at org.dspace.app.xmlui.cocoon.AspectGenerator.setup(AspectGenerator.java:81)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.prepareInternal(AbstractProcessingPipeline.java:480)
        at sun.reflect.GeneratedMethodAccessor329.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy109.prepareInternal(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at org.dspace.app.xmlui.cocoon.AspectGenerator.setup(AspectGenerator.java:81)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.setupPipeline(AbstractCachingProcessingPipeline.java:710)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.prepareInternal(AbstractProcessingPipeline.java:480)
        at sun.reflect.GeneratedMethodAccessor329.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.prepareInternal(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at org.dspace.app.xmlui.cocoon.AspectGenerator.setup(AspectGenerator.java:81)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.setupPipeline(AbstractCachingProcessingPipeline.java:710)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.prepareInternal(AbstractProcessingPipeline.java:480)
        at sun.reflect.GeneratedMethodAccessor329.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.prepareInternal(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at org.dspace.app.xmlui.cocoon.AspectGenerator.setup(AspectGenerator.java:81)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.setupPipeline(AbstractCachingProcessingPipeline.java:710)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.prepareInternal(AbstractProcessingPipeline.java:480)
        at sun.reflect.GeneratedMethodAccessor329.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.prepareInternal(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at org.dspace.app.xmlui.cocoon.AspectGenerator.setup(AspectGenerator.java:81)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.setupPipeline(AbstractCachingProcessingPipeline.java:710)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.prepareInternal(AbstractProcessingPipeline.java:480)
        at sun.reflect.GeneratedMethodAccessor329.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy102.prepareInternal(Unknown Source)
        at org.apache.cocoon.components.source.impl.SitemapSource.init(SitemapSource.java:292)
        at org.apache.cocoon.components.source.impl.SitemapSource.<init>(SitemapSource.java:148)
        at org.apache.cocoon.components.source.impl.SitemapSourceFactory.getSource(SitemapSourceFactory.java:62)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:153)
        at org.apache.cocoon.components.source.CocoonSourceResolver.resolveURI(CocoonSourceResolver.java:183)
        at org.apache.cocoon.generation.FileGenerator.setup(FileGenerator.java:99)
        at sun.reflect.GeneratedMethodAccessor294.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.cocoon.core.container.spring.avalon.PoolableProxyHandler.invoke(PoolableProxyHandler.java:71)
        at com.sun.proxy.$Proxy103.setup(Unknown Source)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.setupPipeline(AbstractProcessingPipeline.java:343)
        at org.apache.cocoon.components.pipeline.impl.AbstractCachingProcessingPipeline.setupPipeline(AbstractCachingProcessingPipeline.java:710)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.preparePipeline(AbstractProcessingPipeline.java:466)
        at org.apache.cocoon.components.pipeline.AbstractProcessingPipeline.process(AbstractProcessingPipeline.java:411)
        ... 85 more

```

--- a/dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/eperson/EditProfile.java
+++ b/dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/eperson/EditProfile.java
@@ -158,11 +158,15 @@ public class EditProfile extends AbstractDSpaceTransformer
 
     private static final Message T_select_collection =
         message("xmlui.EPerson.EditProfile.select_collection");
- 
-    private static final Message T_head_auth =
-        message("xmlui.EPerson.EditProfile.head_auth");
-    
-    //private static final Message T_head_identify =
+
+	private static final Message T_head_auth =
+		message("xmlui.EPerson.EditProfile.head_auth");
+
+	private static final Message T_invalid_item =
+		message("xmlui.EPerson.EditProfile.invalid_item");
+
+
+	//private static final Message T_head_identify =
     //    message("xmlui.EPerson.EditProfile.head_identify");
     
     private static final Message T_head_security =
@@ -492,29 +496,37 @@ public class EditProfile extends AbstractDSpaceTransformer
 
 	                Bitstream bitstream = Bitstream.find(context, bitstreamID);
 	                org.dspace.content.Item item = (org.dspace.content.Item)bitstream.getParentObject();
+					String base = ConfigurationManager.getProperty("dspace.url");
 
-	                String base = ConfigurationManager.getProperty("dspace.url");
-	    			StringBuffer itemLink = new StringBuffer().append(base)
-							  .append(base.endsWith("/") ? "" : "/")
-							  .append("/handle/")
-							  .append(item.getHandle());
+					if ( null != item ) {
+						StringBuffer itemLink = new StringBuffer().append(base)
+							.append(base.endsWith("/") ? "" : "/")
+							.append("/handle/")
+							.append(item.getHandle());
 
-	                r.addCell().addXref(itemLink.toString(), "" + item.getID());
+						r.addCell().addXref(itemLink.toString(), "" + item.getID());
 
-	    			StringBuffer bitstreamLink = new StringBuffer().append(base)
-								  .append(base.endsWith("/") ? "" : "/")
-								  .append("bitstream/handle/")
-								  .append(item.getHandle())
-								  .append("/")
-								  .append(URLEncoder.encode(bitstream.getName(), "UTF8"))
-								  .append("?sequence=").append(bitstream.getSequenceID());
-	                r.addCell().addXref(bitstreamLink.toString(), "" + bitstream.getID());
+						StringBuffer bitstreamLink = new StringBuffer().append(base)
+							.append(base.endsWith("/") ? "" : "/")
+							.append("bitstream/handle/")
+							.append(item.getHandle())
+							.append("/")
+							.append(URLEncoder.encode(bitstream.getName(), "UTF8"))
+							.append("?sequence=").append(bitstream.getSequenceID());
+						r.addCell().addXref(bitstreamLink.toString(), "" + bitstream.getID());
+
+					}else {
+						// something can go wrong
+						r.addCell().addContent( T_invalid_item );
+						r.addCell().addContent( T_invalid_item );
+					}
+
 
 	                Cell c = r.addCell();
 	                java.util.List<UserMetadata> extraMetaData = functionalityManager.getUserMetadata_License(ur.getEpersonId(), license.getTransactionId());
 	                for(UserMetadata metadata : extraMetaData) {
 	                	c.addHighlight("label label-info font_smaller").addContent(metadata.getMetadataKey() + ": " +metadata.getMetadataValue());
-           }
+           			}
            }
            
 			} else {

--- a/dspace-xmlui/src/main/webapp/i18n/messages.xml
+++ b/dspace-xmlui/src/main/webapp/i18n/messages.xml
@@ -542,6 +542,7 @@
 	<message key="xmlui.EPerson.EditProfile.error_invalid_password">Choose a password at least 6 characters long</message>
 	<message key="xmlui.EPerson.EditProfile.error_unconfirmed_password">Please retype your password to confirm</message>
 	<message key="xmlui.EPerson.EditProfile.head_auth">Authorization groups you belong to</message>
+	<message key="xmlui.EPerson.EditProfile.invalid_item">Invalid item</message>
 	<message key="xmlui.EPerson.EditProfile.head_identify">Identify</message>
 	<message key="xmlui.EPerson.EditProfile.head_security">Security</message>
 

